bokeh/models/annotations.py

```diff
@@ -1096,6 +1097,42 @@ class Whisker(Annotation):
 
     level = Override(default="underlay")
 
+class InfoPane(Annotation):
+    ''' Renders a info-pane.
+
+    .. note::
+        This model is currently managed by BokehJS and is not useful
+        directly from python.
+
+    '''
+    level = Override(default="overlay")
+
+    anchor = Enum(TooltipAttachment, help="""
+    Whether the pane should be anchored at the left or right of the pane
+    or above or below it, or if it should be automatically placed
+    in the horizontal or vertical dimension.
+    """)
```

Comment:
Just noting we may also potentially want to allow corners as anchor points, e.g. for cases where users want ot position in a fixed corner of a plot. It's a little unclear about what should happen with the "arrow" in those cases tho, so we can punt on that for now.

Reply:
I will leave it as a TODO comment in the code if that works for now.